logical/auth.go

```diff
@@ -69,6 +71,9 @@ type Auth struct {
 	// mappings groups for the group aliases in identity store. For all the
 	// matching groups, the entity ID of the user will be added.
 	GroupAliases []*Alias `json:"group_aliases" mapstructure:"group_aliases" structs:"group_aliases"`
+
+	// The set of CIDRs that this token can be used with
+	BoundCIDRs []*sockaddr.SockAddrMarshaler `json:"bound_cidrs"`
```

Comment:
This will also need to be added to `logical/plugin/pb/backend.proto` and subsequently to `logical/plugin/translation.go`.

Reply:
Thanks for point that out! I will tackle this after https://github.com/hashicorp/vault/pull/4501 is merged.

Reply:
:heavy_check_mark:

Reply:
Please don't forget about this :-)

Reply:
Oh nm, you already did it.